proxy/http/proxy/server.py

```diff
@@ -360,20 +361,8 @@ def generate_upstream_certificate(
         with self.lock:
             if not os.path.isfile(cert_file_path):
                 logger.debug('Generating certificates %s', cert_file_path)
-                # TODO: Parse subject from certificate
-                # Currently we only set CN= field for generated certificates.
```

Comment:
Do preserve TODOs.  Someday it will be worth addressing these.

Reply:
Done.